src/module/actor/sheets/SR5MatrixActorSheet.ts

```diff
@@ -93,6 +93,82 @@ export class SR5MatrixActorSheet extends SR5BaseActorSheet {
         html.find('.open-matrix-device').on('click', this._onOpenMatrixDevice.bind(this));
 
         html.find('.targets-refresh').on('click', this._onTargetsRefresh.bind(this));
+
+        html.find('.setup-pan').on('click', this._addAllEquippedWirelessDevicesToPAN.bind(this));
+
+        // Matrix Network
+        html.find('.connect-to-network').on('click', this._onConnectToMatrixNetwork.bind(this));
+        // Matrix Target - Connected Icons Visibility Switch
+        html.find('.toggle-connected-matrix-icons').on('click', this._onToggleConnectedMatrixIcons.bind(this));
+    }
+
+    /**
+     * Allow the user to select a matrix network to connect to.
+     */
+    async _onConnectToMatrixNetwork(event) {
+        event.stopPropagation();
+
+        const dialog = new SelectMatrixNetworkDialog(this.document);
+        const network = await dialog.select();
+        if (dialog.canceled) return;
+
+        await this.document.connectNetwork(network);
+        this.render();
+    }
+
+    /**
+     * Within the list of avialable matrix targets, toggle visibility of sub-sections for connected icons
+     * for a single matrix target.
+     *
+     * This is done by clicking on a specific icon by user input and will trigger:
+     * - switching out sheet display
+     * - provide a display of additional matrix icons underneath uuid
+     */
+    async _onToggleConnectedMatrixIcons(event) {
+        event.stopPropagation();
+
+        const uuid = Helpers.listItemUuid(event);
+        if (!uuid) return;
+
+        // Mark main icon as open or closed.
+        if (this._connectedIconsOpenClose[uuid]) delete this._connectedIconsOpenClose[uuid];
+        else this._connectedIconsOpenClose[uuid] = true;
+
+        // Trigger new icons to be shown or hidden.
+        this.render();
+    }
+
+    /**
+     * Add All equipped wireless items on the character to their PAN
+     * @param event
+     */
+    async _addAllEquippedWirelessDevicesToPAN(event) {
```

Comment:
That's a feature that should be added to the 123/marks PR :)
So we don´t forget to mention it on the release changelog.